### DISCUSSION TITLE
Extensions : AdNauseam + Privacy Badger ?

### DISCUSSION THREAD

**Discussion by sysdev00** [dhowe/AdNauseam#2574] [General]
Is it a good idea to use these two extensions? AdNauseam + Privacy Badger
Privacy Badger, reputed by EFF to block trackers, and AdNauseam to click on ad trackers, what do you think?

Have a nice day :)

**Comment by dhowe** [comment]
https://github.com/dhowe/AdNauseam/wiki/FAQ#can-i-combine-adnauseam-with-another-blocker

**Reply by sysdev00** [reply]
Thanks ! Good day :)
